CI failing with

```
/home/runner/work/unison/unison/parser-typechecker/src/Unison/Typechecker/Context.hs:60:1: error: [-Wunused-imports, -Werror=unused-imports]
    The import of ‘Data.Ord’ is redundant
      except perhaps to import instances from ‘Data.Ord’
    To import instances alone, use: import Data.Ord()
   |
60 | import           Data.Ord                       ( comparing )
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

/home/runner/work/unison/unison/parser-typechecker/src/Unison/Typechecker/Context.hs:2279:3: error: [-Wunused-local-binds, -Werror=unused-local-binds]
    Defined but not used: ‘isVar’
     |
2279 |   isVar (Type.Var' _) = True
```

--- a/parser-typechecker/src/Unison/Typechecker/Context.hs
+++ b/parser-typechecker/src/Unison/Typechecker/Context.hs
@@ -57,7 +57,6 @@ import           Data.Functor.Compose           ( Compose(..) )
 import           Data.List
 import           Data.List.NonEmpty             ( NonEmpty )
 import qualified Data.Map                      as Map
-import           Data.Ord                       ( comparing )
 import qualified Data.Sequence                 as Seq
 import           Data.Sequence.NonEmpty         ( NESeq )
 import qualified Data.Sequence.NonEmpty        as NESeq
@@ -2271,9 +2270,6 @@ pruneAbilities want0 have0 = do
       then expandWanted =<< pruneVariables [] pwant
       else pure pwant
   where
-  isVar (Type.Var' _) = True
-  isVar _ = False
-
   isExistential (Type.Var' TypeVar.Existential{}) = True
   isExistential _ = False
 

